packages/frontend/web/browser/boot.ts

```diff
@@ -135,16 +135,15 @@ const onPolyfilled = (): Promise<void> => {
 };
 
 const run = (): void => {
-    /*
-        We want to run `onPolyfilled` only after polyfill.io has initialised
-        By the time this script runs, if `window.guardian.polyfilled` is true,
-        meaning that polyfill.io has initialised, then we run onPolyfilled(), otherwise
-        we stick it in window.guardian.onPolyfilled to be ran later.
-    */
+    // Expects polyfill to have run
     if (window.guardian.polyfilled) {
         onPolyfilled();
     } else {
-        window.guardian.onPolyfilled = onPolyfilled;
+        // (Should never happen)
+        // tslint:disable-next-line:no-console
+        console.log(
```

Comment:
This code runs before everything else so I'm not sure if I can reliably use Raven here?

Reply:
I actually wondered that after I finished the review as I notice we wrap boot in raven. It's a shame as we could really do with knowing if things bail out because of polyfill